icu4c/source/i18n/unicode/timezone.h

```diff
@@ -790,7 +790,7 @@ class U_I18N_API TimeZone : public UObject {
      *           same class ID. Objects of other classes have different class IDs.
      * @stable ICU 2.0
      */
-    virtual UClassID getDynamicClassID(void) const = 0;
+    //virtual UClassID getDynamicClassID(void) const = 0;
```

Comment:
This removal will remove the build/compile time checking of subclass implement getDynamicClassID.

Reply:
virtual UClassID getDynamicClassID(void) const override = 0;